master/reference/advanced/etcd-rbac/kubernetes.md

```diff
@@ -81,5 +80,6 @@ The pieces that would need updating are:
 	- The `etcd-ca` field needs the base64 encoded file contents from the
 	  Certificate Authority certificate.
 
-Once the updates above are made then the manifest can be applied in the normal
-manner.
+- Disable etcd compaction in the kube-controller deployment by setting the `COMPACTION_PERIOD` environment variable to 0.
```

Comment:
I think you should change kube-controller to be calico-kube-controllers since that is the name of the deployment.

Reply:
Also in your comment in the issue that this PR addresses, you mention this should only be done if etcd is shared with K8s which I don't think this section is restricted to, so I think we should include a note about that.

Reply:
I can add a note, but worth noting that the "Why you should follow this guide" above says "If you're sharing an etcd", so I think the intention is that this is limited to that scenario.

Reply:
Ok no need to add the note then unless you think it helps.